KubeArmor/core/kubeUpdate.go

```diff
@@ -44,14 +44,18 @@ func (dm *KubeArmorDaemon) HandleNodeAnnotations(node *tp.Node) {
 	}
 
 	if lsm, err := os.ReadFile("/sys/kernel/security/lsm"); err == nil {
-		if !strings.Contains(string(lsm), "apparmor") && !strings.Contains(string(lsm), "selinux") {
-			// exception: neither AppArmor nor SELinux
+		hasAppArmor := strings.Contains(string(lsm), "apparmor")
+		hasSelinux := strings.Contains(string(lsm), "selinux")
+		hasBPF := strings.Contains(string(lsm), "bpf")
+
+		if !hasBPF && !hasSelinux && !hasAppArmor {
```

Comment:
@tesla59 this logic looks good. :heavy_check_mark: 
Just one more point - in Kubernetes mode, if KubeArmor is installed using operator the node will have a `kubearmor.io/enforcer` annotation which is applied by the operator. This contains the enforcer that has to be used.
So can we have a condition like
Check the `kubearmor.io/enforcer` annotation; if it is present, get enforcer from it else get enforcer using the LSM file. I think we should be good then.

Reply:
added a check for `kubearmor.io/enforcer` annotation